assets/js/DataGrid.js

```diff
@@ -152,7 +156,18 @@ export default class DataGrid extends React.Component {
         minWidth={this.state.gridWidth -2}
         minHeight={this.state.gridHeight-2}   // -2 because grid has borders, don't want to expand our parent DOM node
         enableCellSelect={true}
-        onGridRowsUpdated={this.onGridRowsUpdated} />
+        onGridRowsUpdated={this.onGridRowsUpdated}
+        onGridSort={this.onGridSort}
+        // Ref sets the sort direction on the datagrid
+        ref={(component) => {
```

Comment:
This code should be broken out into a separate function. Also, not sure this is the right place to put this code... but can't think of a better way.